packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -301,6 +304,9 @@ export const MafsGraph = (props: MafsGraphProps) => {
                     {interactionPrompt && (
                         <View
                             style={{
+                                display: interactionPrompt
+                                    ? undefined
+                                    : "hidden",
```

Comment:
Nit: Can you add a brief note in your PR description saying that this change is also in your PR even though it's not the main focus? It's helpful for me to follow PR expectations that way, and I think it's nice to document for posterity.